kafka-client/src/main/java/dev/responsive/kafka/api/async/internals/events/AsyncEvent.java

```diff
@@ -111,7 +111,10 @@ public enum State {
 
   private final Object inputRecordKey;
   private final Object inputRecordValue;
+
+  private final String asyncProcessorName;
```

Comment:
I didn't end up using/needing this in the current PR, but I'm leaving it in since I think it might be useful/necessary later (ie once we expand to allow more than one async processor per topology)